portal-api/src/v1/users/validation/rules/select-at-least-one-bank.js

```diff
@@ -19,13 +19,13 @@ const selectAtLeastOneBank = (user, change) => {
     ];
   }
 
-  const cantHaveAllBank = change.bank === CONSTANT.ALL && !change.roles?.includes(ADMIN);
+  const cantHaveAllBank = change.bank?.name === CONSTANT.ALL && !(change.roles?.includes(ADMIN) || change.roles?.includes(READ_ONLY));
```

Comment:
```suggestion
const eligibleRoles = [ADMIN, READ_ONLY];

const isBankAll = change.bank?.name === CONSTANT.ALL;
const isRoleEligible = change.roles?.includes(eligibleRoles)

const cantHaveAllBank = isBankAll && isRoleValid;
```

Above improves readability.